spec/core/data_structures.md

```diff
@@ -4,28 +4,31 @@ Here we describe the data structures in the Tendermint blockchain and the rules
 
 The Tendermint blockchains consists of a short list of data types:
 
-- [`Block`](#block)
-- [`Header`](#header)
-- [`Version`](#version)
-- [`BlockID`](#blockid)
-- [`PartSetHeader`](#partsetheader)
-- [`Time`](#time)
-- [`Data` (for transactions)](#data)
-- [`Commit`](#commit)
-- [`CommitSig`](#commitsig)
-- [`BlockIDFlag`](#blockidflag)
-- [`Vote`](#vote)
-- [`CanonicalVote`](#canonicalvote)
-- [`SignedMsgType`](#signedmsgtype)
-- [`EvidenceData`](#evidence_data)
-- [`Evidence`](#evidence)
-- [`DuplicateVoteEvidence`](#duplicatevoteevidence)
-- [`LightClientAttackEvidence`](#lightclientattackevidence)
-- [`LightBlock](#lightblock)
-- [`SignedHeader`](#signedheader)
-- [`Validator`](#validator)
-- [`ValidatorSet`](#validatorset)
-- [`Address`](#address)
+- [Data Structures](#data-structures)
+  - [Block](#block)
+  - [Execution](#execution)
+  - [Header](#header)
+  - [Version](#version)
+  - [BlockID](#blockid)
+  - [PartSetHeader](#partsetheader)
+  - [Time](#time)
+  - [Data](#data)
+  - [Commit](#commit)
+  - [CommitSig](#commitsig)
+  - [BlockIDFlag](#blockidflag)
+  - [Vote](#vote)
+  - [CanonicalVote](#canonicalvote)
+  - [SignedMsgType](#signedmsgtype)
+  - [Signature](#signature)
+  - [EvidenceData](#evidencedata)
+  - [Evidence](#evidence)
+    - [DuplicateVoteEvidence](#duplicatevoteevidence)
+    - [LightClientAttackEvidence](#lightclientattackevidence)
+  - [LightBlock](#lightblock)
+  - [SignedHeader](#signedheader)
+  - [ValidatorSet](#validatorset)
+  - [Validator](#validator)
+  - [Address](#address)
```

Comment:
Hmm :thinking: It seems like there is some dark vs magic that is randomly changing the md document

Reply:
it looks better!! Did you use some auto format?

Reply:
must be

Reply:
VSCode has two excellent markdown extensions:

- https://github.com/yzhang-gh/vscode-markdown

Reply:
yes I'm using the one by David Anson but something else is overriding it and creating/formatting a table of contents when there shouldn't be on

Reply:
Ah yeah it was some other extension `Markdown All-in-one` that was causing it